### PR TITLE
Implement Ansible playbook runner with YAML config support

### PR DESCRIPTION
**Pull request by danylevskyi**
So, here is a final version of Ansible playbook runner with YAML config.

What this PR does?

1. Removes `hosts` file. **YAY!** We got rid of one more redundant file.
2. Adds logic to `run.sh` which reads config.yml file. **YAY!** We got **SINGLE** truly YAML configuration file
3. Adds logic to `jenkinsbox.yml` which checks whether the playbook was executed with the runner.

What does it mean?

Now end-users do not need to edit `hosts` and `jenkinsbox.yml` files to start a provisioning. Now end-user configuration could be stored in `config.yml` file only.

How to review:

1. Edit `config.yml` and provide the IP of your server.
2. Execute command `./run.sh` and make your coffee.
3. Get your Jenkins server!

### CODE REVIEW

**Comment by Sanchiz** [comment]
1) If there's no way to pass IP into playbook through command line(or extra-vars), we should add `hosts` to `.gitignore`.
2) We should pass all variable as `extra-vars`.

> So, probably we should pass project name as extra var, but it needs changing playbook slightly.

I'm not sure that we should change playbook, it should work with `extra-vars` out of the box. We just should update `project_name` in `jenkinsbox.yml` and set some value. Maybe `cibox`.

3) How many nested levels does support this YAML parser? E.g. will it work with such variables in `config.yml`:
```
drushrc:
      drush_usage_log: 1
      drush_usage_send: 1
```

**Comment by danylevskyi** [comment]
@Sanchiz 
3) There is no limit for nesting levels (I believe :)). Your example configuration should work.

**Comment by danylevskyi** [comment]
@Sanchiz 
1) I totally agree that we should get rid of inventory file (hosts). There are some links telling that it's possible.

**Comment by danylevskyi** [comment]
@Sanchiz, as a result:
1. We should pass IP like that:
`ansible-playbook -i 177.122.26.33, playbook.yml`
2. We should pass all variables from `config.yml` via `extra-vars`.

And now I have a question:
1. Should we check if `config.yml` has all required variables before running `ansible-playbook`?

**Comment by danylevskyi** [comment]
Theoretically, we could create a function which will loop thorough the full list of items in `config.yml` and pass all them to the `extra-vars`...

File: run.sh [hunk 1]
```diff
@@ -1,3 +1,57 @@
...
+#
+# Getting items:
+# echo $config_item_one
+#
+# To get nested items:
+#
+# echo $config_item_two_sub_item
+#
+# Note: There is "shyaml" python script to read YAML files, but we will probably
+# not use it in order not to have additional dependcies.
```
Comment (podarok) [inline run.sh]: we have ansible installed.
If that dependency can help us to get rid of tough code, that is below - let's do it )

Comment (danylevskyi) [inline run.sh]: It means that the client should install `shyaml` package in order to run `run.sh`.

**Pull request merged**